homeassistant/components/alexa/state_report.py

```diff
@@ -85,8 +85,10 @@ async def async_entity_state_listener(
         if not should_report and not should_doorbell:
             return
 
-        if should_doorbell:
-            if new_state.state == STATE_ON:
+        if should_doorbell and new_state:
```

Comment:
```suggestion
        if should_doorbell:
```

Reply:
You're right, `new_state` can't be `None` due to the guard clause above.